app/custom_domain_validation.py

```diff
@@ -18,21 +25,46 @@ class DomainValidationResult:
 
 class CustomDomainValidation:
     def __init__(
-        self, dkim_domain: str, dns_client: DNSClient = get_network_dns_client()
+        self,
+        dkim_domain: str,
+        dns_client: DNSClient = get_network_dns_client(),
+        partner_domains: Optional[dict[int, str]] = None,
+        partner_domains_validation_prefixes: Optional[dict[int, str]] = None,
     ):
         self.dkim_domain = dkim_domain
         self._dns_client = dns_client
-        self._dkim_records = {
-            f"{key}._domainkey": f"{key}._domainkey.{self.dkim_domain}"
-            for key in ("dkim", "dkim02", "dkim03")
-        }
+        self._partner_domains = partner_domains or PARTNER_DOMAINS
+        self._partner_domain_validation_prefixes = (
+            partner_domains_validation_prefixes or PARTNER_DOMAIN_VALIDATION_PREFIXES
+        )
 
-    def get_dkim_records(self) -> {str: str}:
-        """
-        Get a list of dkim records to set up. It will be
+    def get_ownership_verification_record(self, domain: CustomDomain) -> str:
+        prefix = "sl-verification"
```

Comment:
Maybe having the prefix as `sl` and then forcing the record to have `{prefix}-verification` so that we cannot have a prefix without the `verification` keyword? wdyt?

Reply:
I have no strong feeling, I'll change it 👍